packages/blade/docs/guides/Usage.stories.mdx

```diff
@@ -0,0 +1,64 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Guides/Usage" />
+
+# 👀 How to use blade?
+<br/>
+<br/>
+
+## Tokens
+To start using tokens in your application you can follow below steps:
+1. Wrap you component tree with `BladeProvider` and pass it `paymentTheme` or `bankingTheme` tokens.
+```jsx
+// App entry point
+import { BladeProvider } from '@razorpay/blade/components';
+import { paymentTheme } from '@razorpay/blade/tokens';
+
+function App(): JSX.Element {
+  return (
+    <React.Fragment>
+      <BladeProvider themeTokens={paymentTheme}>
+        <Card />
+      </BladeProvider>
+    </React.Fragment>
+  );
+}
+
+export default App;
+```
+2. You can also pass in optional `colorScheme` prop to the `BladeProvider` mentioning whether you want the `light`, `dark` or `system`'s default color scheme. The default is `light`.
+```jsx
+<BladeProvider themeTokens={paymentTheme} colorScheme='dark'>
+   <Card/>
+</BladeProvider>
+```
+3. After you've wrapped your component tree under `BladeProvider`, you can use the `useTheme()` hook to get the access to the current theme context.
+```jsx
+import { useTheme } from '@razorpay/blade/components';
+
+const Card = (): React.ReactElement => {
+  const { theme } = useTheme();
+  return (
+    <React.Fragment>
+      <DisplayLarge theme={theme}>Cash Advance</DisplayLarge>
+      <StyledCard theme={theme}>
+        <CaptionRegular theme={theme}>
+          This amount will be deducted in 3 installments from your settlement balance between Feb
+          18-20 on a daily basis.
+        </CaptionRegular>
+      </StyledCard>
+    </React.Fragment>
+  );
+};
+
+const StyledCard = styled.div(
+  ({ theme }: { theme: Theme }) => `
+    width: 368px;
+    background-color: ${theme.colors.surface.background.level2.lowContrast};
+    border-radius: ${theme.border.radius.medium}px;
+    padding: ${theme.spacing[5]}px;
```

Comment:
Is this the correct usage for spacing? 🤔

Reply:
yes, until we add the way to automatically attach units